hushline/settings.py

```diff
@@ -704,6 +719,38 @@ def update_smtp_settings() -> Response | str:
         flash("👍 SMTP settings updated successfully")
         return redirect(url_for(".index"))
 
+    @bp.route("/update-brand-primary-color", methods=["POST"])
+    @admin_authentication_required
+    def update_brand_primary_color() -> Response | str:
+        if (host_org := db.session.get(HostOrganization, 1)) is None:
```

Comment:
Use of a magic number. Should be a constant in the `HostOrganization` object.

Reply:
Addressed by (https://github.com/scidsg/hushline/pull/573/commits/007cf1b1688684d8cf8751d28d8f4537788fb390)